jax/numpy/lax_numpy.py

```diff
@@ -1967,7 +1967,34 @@ def _pad_edge(array, pad_width):
 def _pad(array, pad_width, mode, constant_values):
   array = asarray(array)
   nd = ndim(array)
-  pad_width = np.broadcast_to(np.asarray(pad_width), (nd, 2))
```

Comment:
I'm unsure why we didn't just convert the output of the original computation to a tuple of tuples...

Reply:
You mean, why have the if/else chain and why not just do `tuple(map(tuple, np.broadcast_to(...)))`?

I can't remember now, but I think I tried that first and something went wrong. I'll check again and revise it if it works!